docs/references/chrome-extension/sync-host.mdx

```diff
@@ -0,0 +1,132 @@
+      CLERK_FRONTEND_API={{fapi_url}}
+      PLASMO_PUBLIC_SYNC_HOST=http://localhost
+      ```
+    </Tab>
+
+    <Tab>
+      Add `PLASMO_PUBLIC_SYNC_HOST` to your `.env.production` file. The value should be the domain your web app's production server runs on. For example, `https://clerk.com`.
+
+      ```env {{ filename: '.env.production', mark: [3] }}
+      PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+      CLERK_FRONTEND_API={{fapi_url}}
+      PLASMO_PUBLIC_SYNC_HOST=https://yourdomain.com
+      ```
+    </Tab>
+  </Tabs>
+
+  ### Add `syncHost` prop to your `<ClerkProvider>`
+
+  Add the `syncHost` prop to your Chrome Extension's `<ClerkProvider>` component. This prop tells the `<ClerkProvider>` which host to sync with.
+
+  ```tsx {{ filename: 'src/popup/layouts/root-layout.tsx', mark: [5, [7, 11], 22] }}
+  import { ClerkProvider, SignedIn, SignedOut, UserButton } from '@clerk/chrome-extension'
+  import { Link, Outlet, useNavigate } from 'react-router-dom'
+
+  const PUBLISHABLE_KEY = process.env.PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY
+  const SYNC_HOST = process.env.PLASMO_PUBLIC_SYNC_HOST
+
+  if (!PUBLISHABLE_KEY || !SYNC_HOST) {
+    throw new Error(
+      'Please add the PLASMO_PUBLIC_CLERK_PUBLISHABLE_KEY and PLASMO_PUBLIC_SYNC_HOST to the .env.development file',
+    )
+  }
+
+  export const RootLayout = () => {
+    const navigate = useNavigate()
+
+    return (
+      <ClerkProvider
+        routerPush={(to) => navigate(to)}
+        routerReplace={(to) => navigate(to, { replace: true })}
+        publishableKey={PUBLISHABLE_KEY}
+        afterSignOutUrl="/"
+        syncHost={SYNC_HOST}
+      >
+        <div className="plasmo-w-[785px] plasmo-h-[600px]">
+          <main>
+            <Outlet />
+          </main>
+          <footer>
+            <SignedIn>
+              <Link to="/settings">Settings</Link>
+              <UserButton />
+            </SignedIn>
+            <SignedOut>
+              <Link to="/">Home</Link>
+              <Link to="/sign-in">Sign In</Link>
+              <Link to="/sign-up">Sign Up</Link>
+            </SignedOut>
+          </footer>
+        </div>
+      </ClerkProvider>
+    )
+  }
+  ```
+
+  ### Configure `host_permissions`
+
+  `host_permissions` specifies which hosts, or websites, will have permission to sync auth state with your app. It accepts an array, allowing you to add more than one host.
+
+  In the `package.json` file, in the `manifest` object, update the `host_permissions` array. Remove `http://localhost/*` and replace with `$PLASMO_PUBLIC_SYNC_HOST/*`, as shown in the following example:
+
+  ```json {{ filename: 'package.json', mark: [6] }}
+  {
+    // The rest of your package.json file
+    "manifest": {
+      "key": "$CRX_PUBLIC_KEY",
+      "permissions": ["cookies", "storage"],
+      "host_permissions": ["$PLASMO_PUBLIC_SYNC_HOST/*", "$CLERK_FRONTEND_API/*"]
+    }
+  }
+  ```
+
+  ### Add the Extension's ID to your web app's `allowed_origins`
```

Comment:
When setting the `.env` values above there were development and production sections but here there is only development. We should drop production above or add production here. That said, adding it here would have the same issue with the secret key here as the Publishable key above.

Reply:
I've edited the instructions below, let me know what you think.
This will have to do until we support production keys in the docs.

Reply:
Looks good!